Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN pip install -r requirements/production.txt
 
 
 # Stage 2 - build frontend
-FROM node:24-bookworm-slim AS frontend-build
+FROM node:24-trixie-slim AS frontend-build
```

Comment:
I think this one needs to be updated in the Objects API dockerfile as well: https://github.com/maykinmedia/objects-api/blob/master/Dockerfile#L23